knowage-vue/src/modules/managers/usersManagement/UserAttributesTab/UserAttributesForm.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <Card style="width: 100%; margin-bottom: 2em">
```

Comment:
do not use inline styles, use primevue [grid columns](https://primefaces.org/primevue/showcase/#/grid) 
this will allow us to create responsive column management when needed.